README.md

```diff
@@ -0,0 +1,25 @@
+# rl-js-agents (add badges here)
+
+Code for [reinforcement learning](https://en.wikipedia.org/wiki/Reinforcement_learning) agents, written in JavaScript.
+
+## Getting Started
+
+This is a [`node`](https://nodejs.org) module that will be made available via GitHub and [`npm`](https://www.npmjs.com/).
+
+Check dependencies and `engines`, etc. in `package.json`; this module targets the current LTS, and possibly higher versions.
+
+Since nothing has been published to `npm` so far, to start hacking: clone this repo with `git` and therein run `npm install`.
+
+Use `npm test` to run the standard (fast) test suite; `npm run report` will generate a comprehensive test artifact.
```

Comment:
Right now, `npm test` just points to `npm run report`. Add `"test": "npm run jest",` or whatever, to taste.